test/api/v1/routes/exercises_filtering_test.rb

```diff
@@ -57,7 +57,7 @@ def test_that_a_track_that_does_not_exist_returns_empty
   private
 
   def getting(url)
-    VCR.use_cassette('exercism_api_current_exercises_v2') do
```

Comment:
I've thought about this a bit. I could make the argument either way.

Your argument is that "deleting stuff is different from renaming stuff". That's one way of looking at it.

The other perspective is that this is removing unneeded things. Renaming is part of removing it, otherwise the removal is incomplete.

I don't think that "one cohesive change" needs to mean that you are doing one type of action. That's like saying that "potatoes" is one commit and "butter" is a second. But I don't eat potatoes without butter. They're one dish.